trestle/core/commands/author/jinja.py

```diff
@@ -175,6 +186,71 @@ def jinja_ify(
             lut['control_io_writer'] = ControlIOWriter()
             lut['ssp_md_writer'] = ssp_writer
 
+            output = JinjaCmd.render_template(template, lut, template_folder)
+
+            output_file = trestle_root / r_output_file
+            if number_captions:
+                output_file.open('w', encoding=const.FILE_ENCODING).write(_number_captions(output))
+            else:
+                output_file.open('w', encoding=const.FILE_ENCODING).write(output)
+
+            return CmdReturnCodes.SUCCESS.value
+
+    @staticmethod
+    def jinja_multiple_md(
+        trestle_root: pathlib.Path,
+        r_input_file: pathlib.Path,
+        r_output_file: pathlib.Path,
+        profile: Optional[str],
+        lut: Dict[str, Any],
+        parameters_formatting: Optional[str] = None
+    ) -> int:
+        """Output profile as multiple markdown files using Jinja."""
+        template_folder = pathlib.Path.cwd()
+
+        # Output to multiple markdown files
+        _, profile_path = ModelUtils.load_top_level_model(trestle_root, profile, Profile)
+        profile_resolver = ProfileResolver()
+        resolved_catalog = profile_resolver.get_resolved_profile_catalog(
+            trestle_root, profile_path, False, False, parameters_formatting
+        )
+        catalog_interface = CatalogInterface(resolved_catalog)
+
+        # Generate a single markdown page for each control per each group
+        for control in catalog_interface.get_all_controls_from_catalog(True):
+            _, group_title, _ = catalog_interface.get_group_info_by_control(control.id)
+            # control could be in sub-group of group so build path to it
+            group_dir = r_output_file
```

Comment:
Here please use the flow in my recent changes to the jinja template.  Get sorted list of groups and in each group call get_sorted_controls_in_group